os/adding-swap.md

```diff
@@ -0,0 +1,95 @@
+# Adding swap in CoreOS
+
+Swap is the process of moving pages of memory to a designated part of the hard disk, freeing up space when needed. Swap can be used to alleviate problems with low-memory environments.
+
+By default CoreOS does not include a partition for swap, however one can configure their system to have swap.
+
+## Managing swap with systemd
+
+A systemd-managed service can be used to create, enable, and disable a swapfile on your system. Write, enable, and start the systemd unit file found in the next section to automatically add swap to a CoreOS node.
+
+### Creating the systemd unit file
+
+The following systemd unit file creates, enable, and disables a 1GiB swapfile. It should be written to `/etc/systemd/system/swap.service`.
```

Comment:
"The following systemd unit creates and enables a service to manage a 1GiB swapfile."